app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -13,5 +13,13 @@ def azure_ad
     def failure
       throw(:warden, recall: 'Errors#forbidden', message: :forbidden)
     end
+
+    # Override the #passthru action. It is used when a GET request is made
+    # to the user auth url. Ideally the GET route would not be added by Devise.
+    # The fix for this is in Devise but awaiting release:
+    # https://github.com/heartcombo/devise/pull/5508
```

Comment:
Looks like they release fairly often - shall we add a slack reminder in on the dev channel to check this in say a month so that we don't forget it?

Reply:
I have an alert set :). Watched kettle never boils and all that :)